### PR TITLE
CHANGELOG scheme fix

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/22908#event-3146423025

## Description
see in referenced issue(bottom comment)

### CODE REVIEW

File: demisto_sdk/commands/common/constants.py [hunk 1]
```diff
@@ -479,22 +479,42 @@ def invalid_v2_script_name(file_path):
...
+PLAYBOOK_CHANGELOG_REGEX = r'{}(?!Test){}/*_CHANGELOG.md$'.format(CAN_START_WITH_DOT_SLASH, PLAYBOOKS_DIR)
+
 TEST_PLAYBOOK_REGEX = r'{}{}/playbook-.*\.yml$'.format(CAN_START_WITH_DOT_SLASH, TEST_PLAYBOOKS_DIR)
 TEST_NOT_PLAYBOOK_REGEX = r'{}{}/(?!playbook).*-.*\.yml$'.format(CAN_START_WITH_DOT_SLASH, TEST_PLAYBOOKS_DIR)
 
 INCIDENT_TYPE_REGEX = r'{}{}/incidenttype-.*\.json$'.format(CAN_START_WITH_DOT_SLASH, INCIDENT_TYPES_DIR)
+INCIDENT_TYPE_CHANGELOG_REGEX = r'{}{}/*_CHANGELOG.md$'.format(CAN_START_WITH_DOT_SLASH, INCIDENT_TYPES_DIR)
+
 INDICATOR_FIELDS_REGEX = r'{}{}/incidentfield-.*\.json$'.format(CAN_START_WITH_DOT_SLASH, INDICATOR_FIELDS_DIR)
+INDICATOR_FIELDS_CHANGELOG_REGEX = r'{}{}/*_CHANGELOG.md$'.format(CAN_START_WITH_DOT_SLASH, INDICATOR_FIELDS_DIR)
```
Comment (yaakovi) [inline demisto_sdk/commands/common/constants.py]:
```suggestion
INDICATOR_FIELD_CHANGELOG_REGEX = r'{}{}/*_CHANGELOG.md$'.format(CAN_START_WITH_DOT_SLASH, INDICATOR_FIELDS_DIR)
```

File: demisto_sdk/commands/common/constants.py [hunk 2]
```diff
@@ -479,22 +479,42 @@ def invalid_v2_script_name(file_path):
...
 REPORT_REGEX = r'{}{}.*report-.*\.json$'.format(CAN_START_WITH_DOT_SLASH, REPORTS_DIR)
+REPORT_CHANGELOG_REGEX = r'{}{}.*_CHANGELOG.md$'.format(CAN_START_WITH_DOT_SLASH, REPORTS_DIR)
+
+MISC_REGEX = r'{}{}.*reputations\.json$'.format(CAN_START_WITH_DOT_SLASH, MISC_DIR)
 MISC_REPUTATIONS_REGEX = r'{}{}.reputations.json$'.format(CAN_START_WITH_DOT_SLASH, MISC_DIR)
-README_REGEX = r'{}{}/readme.*\.md$'.format(CAN_START_WITH_DOT_SLASH, INDICATOR_FIELDS_DIR)
+REPUTATION_REGEX = r'{}{}/reputation-.*\.json$'.format(CAN_START_WITH_DOT_SLASH, MISC_DIR)
+REPUTATION_CHANGELOG_REGEX = r'{}{}/*_CHANGELOG.md$'.format(CAN_START_WITH_DOT_SLASH, MISC_DIR)
+
+README_REGEX = r'{}{}/*_README.*\.md$'.format(CAN_START_WITH_DOT_SLASH, INDICATOR_FIELDS_DIR)
```
Comment (yaakovi) [inline demisto_sdk/commands/common/constants.py]: ?

File: demisto_sdk/commands/common/constants.py [hunk 3]
```diff
@@ -904,7 +924,10 @@ class PB_Status:
     'incidenttype': [INCIDENT_TYPE_REGEX],
     'image': [IMAGE_REGEX],
     'reputation': [REPUTATION_REGEX],
-    'changelog': [INTEGRATION_CHANGELOG_REGEX, PACKS_CHANGELOG_REGEX],
+    'changelog': [INTEGRATION_CHANGELOG_REGEX, PACKS_CHANGELOG_REGEX, REPUTATION_CHANGELOG_REGEX,
+                  INCIDENT_TYPE_CHANGELOG_REGEX, INCIDENT_FIELD_CHANGELOG_REGEX, INDICATOR_FIELDS_CHANGELOG_REGEX,
```
Comment (yaakovi) [inline demisto_sdk/commands/common/constants.py]:
```suggestion
                  INCIDENT_TYPE_CHANGELOG_REGEX, INCIDENT_FIELD_CHANGELOG_REGEX, INDICATOR_FIELD_CHANGELOG_REGEX,
```

File: demisto_sdk/commands/common/constants.py (hunk 2 above)
Comment (bakatzir) [reply demisto_sdk/commands/common/constants.py]: line 517 is faulty(and not being used)
yana will add a proper readme for old JS integrations in: https://github.com/demisto/demisto-sdk/pull/275

Comment (yaakovi) [reply demisto_sdk/commands/common/constants.py]: this is line 517, if it's faulty and not used - remove it

**Comment by bakatzir** [comment]
Also catching the other content types, checked on incidentfields.
without changelog addition:
```
(py37) ➜  content git:(changelog_test) ✗ demisto-sdk validate -g
You are using demisto-sdk 0.4.5.dev13, however version 0.4.4 is available.
You should consider upgrading via "pip3 install --upgrade demisto-sdk" command.
Using git
Running validation on branch changelog_test
Starting validating files structure
Validates only committed files
Validating IncidentFields/incidentfield-Account_ID.json
Missing release notes for: IncidentFields/incidentfield-Account_ID.json
Starting validation against origin/master
Validating IncidentFields/incidentfield-Account_ID.json
Missing release notes for: IncidentFields/incidentfield-Account_ID.json
The files were found as invalid, the exact error message can be located above
```

**Pull request merged**